lib/routes.js

```diff
@@ -22,6 +22,24 @@ const routeMap = {
     OPTIONS: routeOPTIONS,
 };
 
+// Queries supported by AWS that we do not currently support.
+const unsupportedQueries = [
```

Comment:
Suggestion: Why not exporting `unsupportedQueries` and then importing it into the different file when needed.